views/js/qtiCreator/widgets/interactions/Widget.js

```diff
@@ -85,9 +86,9 @@ define([
      * Create a toolbar
      */
     InteractionWidget.createToolbar = function(options){
-
+        const getQtiClassTitle = qtiClass => classTitles[qtiClass] || qtiClass;
```

Comment:
I didn't want to repeat myself, but apparently I need to.

Is it an `es6` file?
I believe we shouldn't mix `es5` and `6` syntax in a single file.